orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -179,27 +172,29 @@ def _create_training_session(self):
             providers = ["CPUExecutionProvider"]
             provider_options = [{}]
 
-        self._forward_session = onnxruntime.InferenceSession(
-            self._onnx_forward.SerializeToString(), providers=providers, provider_options=provider_options)
-        self._backward_session = onnxruntime.InferenceSession(
-            self._onnx_backward.SerializeToString(), providers=providers, provider_options=provider_options)
+        session_options = onnxruntime.SessionOptions()
+        session_options.enable_mem_pattern = False
+        session_options.use_deterministic_compute = False
+        # 0:Verbose, 1:Info, 2:Warning. 3:Error, 4:Fatal. Default is 2.
+        session_options.log_severity_level = 1
```

Comment:
Probably level 1 is not intended in production

Reply:
Updated to 2.

Reply:
Was this change missed?